tests/test_branch_gen_answers_with_custom_http_client.py

```diff
@@ -0,0 +1,16 @@
+# Third Party
```

Comment:
Debating whether this is worth a new file.  These tests still aren't really tests as much as a way to try out the functionality.  I think I would vote for just making this change in the existing "test".

Reply:
My current thinking is we are covering the functional testing pretty well today through the e2e testing.  There are some exceptions so adding functional tests will have some value.  But I would put the priority on adding unit tests.  I think that's the biggest gap and bang for the buck.

Reply:
I started working on the unit/functional test enablement here:

https://github.com/instructlab/eval/pull/165

If we do want to keep this file, note that it should move to scripts/

Reply:
Moved the "test" in tests/test_branch_gen_answers.py